app/javascript/shared/components/ui/LabelDropdown.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div class="dropdown-search-wrap">
+    <h4 class="text-block-title">
+      {{ $t('CONTACT_PANEL.LABELS.LABEL_SELECT.TITLE') }}
+    </h4>
+    <div class="search-wrap">
+      <input
+        ref="searchbar"
+        v-model="search"
+        type="text"
+        class="search-input"
+        autofocus="true"
+        placeholder="Filter"
+      />
+    </div>
+    <div class="list-wrap">
+      <div class="list">
+        <woot-dropdown-menu>
+          <woot-dropdown-item
+            v-for="label in filteredActiveLabels"
+            :key="label.title"
+          >
+            <div class="name-label-icon-wrap">
+              <button class="button clear" @click="() => onAddRemove(label)">
+                <div class="name-label-wrap">
+                  <div
+                    v-if="label.color"
+                    class="label-color--display"
+                    :style="{ backgroundColor: label.color }"
+                  />
+                  <span>{{ label.title }}</span>
+                </div>
+                <i
+                  v-if="selectedLabels.includes(label.title)"
+                  class="icon ion-checkmark-round"
+                />
+              </button>
```

Comment:
I guess this can be a label component with 2 variants.

1. Label which shows the text.
2. Label which shows the text and the close button.

Reply:
Now that the storybook is merged, it will be a great place to start building components.

Reply:
Yeah sure.